android/app/src/main/java/com/reactnativenavigation/react/NavigationReactGateway.java

```diff
@@ -90,6 +90,7 @@ public void onJsDevReload() {
 
     private ReactInstanceManager createReactInstanceManager() {
         ReactInstanceManager.Builder builder = ReactInstanceManager.builder()
+                .setUseOldBridge(true)
```

Comment:
Hiya @guyca, I've tried editing this both locally on my machine and in GitHub, but can't seem to see this line in the file that is referenced above: `android/app/src/main/java/com/reactnativenavigation/react/NavigationReactGateway.java` am I being an idiot? :P